### ISSUE TITLE
Mining address

### ISSUE THREAD

**Issue by ssk666** [UnamSanctam/SilentXMRMiner#451]
I can't find a mining address. Nanopool XMR or Ethermine can't find my rig. I have already tested on 2 machines. I'd been waiting for 1 hour for give opportunity to pool to find me. Cam u help me pls? Thank u

**Comment by UnamSanctam** [comment]
Can you post your settings? Mainly the "Main" page?

**Comment by ssk666** [comment]
![Screenshot_3](https://user-images.githubusercontent.com/78388995/135731615-4ee565d6-32c4-40b0-96d8-06e4308352a7.png)

**Comment by ssk666** [comment]
So, it's a recieve address. Or i need an ID miner

**Comment by UnamSanctam** [comment]
No that is correct, and you get no hashrate on nanopool? Also remember to enable "SSL/TLS" in the "Mining" tab since you use the SSL port.

**Comment by UnamSanctam** [comment]
That also isn't the latest version of the miner just so you know.

**Comment by ssk666** [comment]
I downloaded a new version of xmr but i have the same problem with eth. I don't understand what i gotta enter here (check screenshots). I tried with SSL/TLS and 'standart' with no results
![xmr](https://user-images.githubusercontent.com/78388995/135731727-697a2bd1-ad72-4d82-abb6-acbe00ab3b84.png)
![eth](https://user-images.githubusercontent.com/78388995/135731728-55f46863-6566-4bcc-9adf-76a0b3119419.png)

**Comment by UnamSanctam** [comment]
Enter your wallet address there and press search.

**Comment by ssk666** [comment]
wallet address is a recieve address or what?

**Comment by UnamSanctam** [comment]
Yes exactly, your Monero or Ethereum wallet address.

**Comment by ssk666** [comment]
![xmr res](https://user-images.githubusercontent.com/78388995/135731830-8ed6fbb3-91f5-4203-96b5-5f96995be468.png)
![Screenshot_4](https://user-images.githubusercontent.com/78388995/135731832-e60702b7-ce39-4b7d-9d25-cfabf35f173b.png)
Now my miners are not active but when they were active there was the same result

**Comment by UnamSanctam** [comment]
Screenshot the "Mining" tab as well.

**Comment by ssk666** [comment]
![ethm](https://user-images.githubusercontent.com/78388995/135731935-8891b3ef-acb2-40f0-8d23-d1e985ba5f58.png)
![Screenshot_5](https://user-images.githubusercontent.com/78388995/135731936-a06a684f-80b0-45d5-81a3-796d94acbf0d.png)
"on the pools until you have mined at least 1 share" yeah, i saw this message but didn't understand what does it mean

**Comment by ssk666** [comment]
![ethm](https://user-images.githubusercontent.com/78388995/135732056-d57580fd-76f6-45a4-8863-7acec9222cfe.png)
![Screenshot_5](https://user-images.githubusercontent.com/78388995/135732057-ac0719ad-ce05-458d-9b7e-6e10b04f54a9.png)

**Comment by ssk666** [comment]
Look!

**Comment by UnamSanctam** [comment]
Yes those should work, you now need to mine to both pools until you find a share (completing a job sent by the pool) for it to appear on the pools. What GPU do you have?

**Comment by ssk666** [comment]
2060 super

**Comment by ssk666** [comment]
I can launch both miners and check result in the morning. U saw how i entered an addresses in the pools, it's alright?

**Comment by UnamSanctam** [comment]
Yes that is correct.

Your GPU should work for Ethereum, so if you start both miners then they should eventually appear on the pools (pool statistic are never instant since that would take a lot of performance). After you have started the miners (and they have injected and everything) then you can run [MinerCheck3.zip](https://github.com/UnamSanctam/SilentXMRMiner/files/7272253/MinerCheck3.zip) and you should see if the miners are running under the "Miners:" text.

**Comment by ssk666** [comment]
Okey! Thank u so much!

**Comment by ssk666** [comment]
![Screenshot_3](https://user-images.githubusercontent.com/78388995/135732341-ba125d12-9dc1-4c42-94ef-353f30d858de.png)
I tried execute a nicehash miner and ur programm doesn't see it

**Comment by ssk666** [comment]
Bro look pls!

**Comment by ssk666** [comment]
I launched both ur miners and ur programm doesn't see its

**Comment by UnamSanctam** [comment]
You started both of the miners you built?

**Comment by ssk666** [comment]
Yeah from screenshots above👆

**Comment by UnamSanctam** [comment]
Hmm, then they should appear there unless you have a 32-bit OS or something. Can you send the files you built here?

**Comment by ssk666** [comment]
U mean .exe builders? I have 64-bit OS

**Comment by UnamSanctam** [comment]
No the two miners you built using the builders.

**Comment by ssk666** [comment]
[ready miners.zip](https://github.com/UnamSanctam/SilentXMRMiner/files/7272314/ready.miners.zip)
this?

**Comment by UnamSanctam** [comment]
Yes those, they work fine for me. Are you sure you started them and then waited a bit until running the MinerCheck.exe?

**Comment by ssk666** [comment]
Oh... I'm so confused because now i checked miners and checker see them! Sorry for this

**Comment by ssk666** [comment]
Thank you for your help and for your time!

**Comment by UnamSanctam** [comment]
No problem, and no worries, they might have been still installing or waiting when you checked first.

**Comment by ssk666** [comment]
Can u give me smth for a private messaging?

**Comment by UnamSanctam** [comment]
You can contact me on Discord: Unam Sanctam#3135
